src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -142,15 +143,19 @@ export default {
 				if (!this.allFacets) this.allFacets = await fetchLoanFacets(this.apollo);
 
 				// Get filtered facet options from FLSS
+				// TODO: Prevent this from running on every query
 				const { isoCodes, themes } = await runFacetsQueries(this.apollo, data?.loanSearchState);
```

Comment:
@dyersituations This method is currently called with every loan query. A future optimization would be to structure the data so we could check for changes that require updates to the facet counts. ie. Sort and Paging updates wouldn't require an update.

Reply:
No big rush on this one, we'll make a ticket for later